lib/streamlit/elements/widgets/button_group.py

```diff
@@ -156,47 +212,70 @@ def _build_proto(
     proto.form_id = current_form_id
     proto.disabled = disabled
     proto.click_mode = click_mode
+    proto.style = ButtonGroupProto.Style.Value(style.upper())
+
+    if label is not None:
+        proto.label = label
+        proto.label_visibility.value = get_label_visibility_proto_value(
```

Comment:
How does it look visually if label is `None`?

Reply:
The spec does not officially allow for the label to be `None`. However, if you still pass it like this is looks like `label_visibility` is set to `hidden`

Reply:
Maybe add a quick comment about that here.

Reply:
Done!